### ISSUE TITLE
Rust CI builds for multiple targets

### ISSUE THREAD

**Issue by Michael-F-Bryan** [hotg-ai/librunecoral#19]
In #18 we added support for cross-compiling `librunecoral` to Android. This project primarily exists to give Rune's Rust code access to TensorFlow Lite, so we should make sure CI can cross-compile the Rust bindings to the various targets we support.

I haven't tested it, but we'll probably need to switch the Rust build to something like this:

```yaml
name: Rust bindings for ${{ matrix.target }}
strategy:
  matrix:
    include:
      - os: ubuntu-latest
        target: x86_64-unknown-linux-gnu
      - os: ubuntu-latest
        target: aarch64-unknown-linux-gnu
      # TODO: add MacOS, iOS, and Windows builds to the matrix
steps:
- ...
- name: Install Rust
  uses: actions-rs/toolchain@v1
  with:
    profile: minimal
    toolchain: stable
    override: true
    target: ${{ matrix.target }}
- name: Check
  uses: actions-rs/cargo@v1
  with:
    command: check
    args: --verbose --manifest-path rust/Cargo.toml --target ${{ matrix.target }}
- name: Test
  uses: actions-rs/cargo@v1
  with:
    command: test
    args: --verbose --manifest-path rust/Cargo.toml --target ${{ matrix.target }}
```

**Comment by saidinesh5** [comment]
I think we can move towards this by providing a `LIBRUNECORAL_DIR`  environment variable to provide prebuilt runecoral static library as per the platform, so we won't have to modify the cross containers too much to achieve what we want.

**Comment by Michael-F-Bryan** [comment]
That's a good idea. We could modify the build script to look for `$LIBRUNECORAL_DIR` and add that to the library search path, falling back to compiling from source when it isn't present.

I believe that's the same strategy the LLVM bindings use.

**Comment by saidinesh5** [comment]
Fixed by #35
